.github/dependabot.yml

```
@@ -3,8 +3,13 @@
 
 version: 2
 
-updates:
+registries:
+  npm-github:
+    type: npm-registry
+    url: npm.pkg.github.com
+    token: ${{secrets.GH_REGISTRY_ACCESS_TOKEN}}
```

From what I see the secret `GH_REGISTRY_ACCESS_TOKEN` does not exist, but `GPR_TOKEN` does.

```suggestion
    token: ${{secrets.GPR_TOKEN}}
```

---

@GreatWizard ?

---

GH_REGISTRY_ACCESS_TOKEN exists in the context of dependabot at the organization level